_sass/minima/custom-styles.scss

```diff
@@ -0,0 +1,334 @@
+// Placeholder to allow defining custom styles that override everything else.
```

Comment:
Could you sort properties by group ? Ex position, display - box-model, font, decoration (background, border-radius, etc)